### PR TITLE
Remove key value pairs contract from base contracts

### PR DESCRIPTION
**Pull request by adamgall**
Please review and merge https://github.com/decentdao/fractal-interface/pull/1645 first.

### CODE REVIEW

File: src/models/DaoTxBuilder.ts [hunk 1]
```diff
@@ -42,6 +45,7 @@ export class DaoTxBuilder extends BaseTxBuilder {
     createSafeTx: SafeTransaction,
     safeContract: GnosisSafeL2,
     txBuilderFactory: TxBuilderFactory,
+    keyValuePairsAddress: string,
```
Comment (Da-Colon) [inline src/models/DaoTxBuilder.ts:48]: I assume eventually these will be updated to the `Address` type

**Review by Da-Colon — APPROVED** [review]
One comment, but approved otherwise

File: src/models/DaoTxBuilder.ts (hunk 1 above)
Comment (adamgall) [reply src/models/DaoTxBuilder.ts:48]: Oh yeah, I've been a bit sloppy in these builder classes with Address/string types.

My intention is that, once this current push to get TypeChain OUTTA HERE is finished, I'll go back through and grep the codebase for `getAddress`, and one by one, remove them, changing the type of the `string`s that they're wrapping with `Address`, fixing whatever that breaks down the stack with the same approach, until eventually all of the things that should be `Addresses` actually are, and we will have minimal/no `getAddress` wrappers anymore.

Comment (Da-Colon) [reply src/models/DaoTxBuilder.ts:48]: ah cool cool, ignore my comment in a few PRs up lol

**Review by DarksightKellar — APPROVED** [review]

**Pull request merged**